modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsController.scala

```diff
@@ -356,9 +357,8 @@ object TcsController {
   @Lenses
   final case class GuidersConfig(
     pwfs1: GuiderConfig@@P1Config,
-    pwfs2: GuiderConfig@@P2Config,
-    oiwfs: GuiderConfig@@OIConfig,
-    aowfs: Boolean@@AOGuide
+    pwfs2OrAowfs: Either[GuiderConfig@@P2Config, ProbeTrackingConfig@@AoGuide],
```

Comment:
Not the best option. There is code dealing with this `Either` all over the place.

Reply:
I am not sure. I still need to remember the base class, because that is what is really used, and I must be forced to explicitly set the tag when passing the value to make sure I am using it in the right place.